src/doc/contrib/src/implementation/schemas.md

```diff
@@ -0,0 +1,29 @@
+# Data Schemas
```

Comment:
Should we have an overview of what this "data schemas" refer to?

Reply:
I've added some context